### DISCUSSION TITLE
When will there be a new release?

### DISCUSSION THREAD

**Discussion by raffomania** [containers/podman-compose#497] [General]
In setting up some projects with podman compose, I've come across many issues that seem to be resolved on the current devel branch.

When are you planning to release these fixes? I'd like to continue using podman-compose through my distro's stable repositories.

**Comment by chris-ds-dk** [comment]
Hi. I would appreciate a new release. It appears that [Issue 586](https://github.com/containers/podman-compose/issues/586) was fixed by [commit 9a08f85](https://github.com/containers/podman-compose/commit/9a08f85ffd08d84796af5e9d104c0b83c5490d81), which has not yet been released.

**Comment by rjeffman** [comment]
I'm resuscitating this old discussion, as it would be nice to have a new release. Is there an expected cadence for the releases?

For my use case, there are some new features I've been using and would be nice to point people to a specific minimum version.

**Comment by p12tic** [comment]
New version has been released since this discussion was opened.
